src/mount_efs/__init__.py

```diff
@@ -673,17 +682,20 @@ def main():
 
     config = read_config()
     bootstrap_logging(config)
-
     fs_id, path, mountpoint, options = parse_arguments(config)
```

Comment:
`fs_id` here now may actually be an IP address, which is confusing. I think it would be better to return one or the other as separate variables. So `parse_arguments` would return either `fs_id, None, mountpoint, options` or `None, ip_addr, mountpoint, options`.